recipes/pep562/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pep562" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pep562-{{ version }}.tar.gz
+  sha256: 58cb1cc9ee63d93e62b4905a50357618d526d289919814bea1f0da8f53b79395
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
```

Comment:
Do we want to add a <3.7 here?

Reply:
I was debating about that.  Upstream says it works for >=3.7 but isn't recommended.  So I think you're right.